src/wsdl/elements.ts

```diff
@@ -802,8 +802,11 @@ export class TypesElement extends Element {
   // fix#325
   public addChild(child) {
     assert(child instanceof SchemaElement);
-
-    const targetNamespace = child.$targetNamespace;
+    const childInclude = child.includes.find((e) => {
+      return e.hasOwnProperty('namespace');
+    });
+    const targetNamespace = child.$targetNamespace ||
+      typeof childInclude !== 'undefined' ? childInclude.namespace : childInclude;
```

Comment:
can you add parens around the ternary?  i think you mean this:

```
    const targetNamespace = child.$targetNamespace ||
      (typeof childInclude !== 'undefined' ? childInclude.namespace : childInclude);
```

also, the tests are failing because of a TypeError when trying to resolve `.namespace`.  please see the travis output in the builds.

Reply:
I was so invested on fixing the first test, I didn't stop to see my attempt at fixing the lint errors broke the logic.